documentation/polymer-templates/tutorial-polymer-template-basic.asciidoc

```diff
@@ -0,0 +1,74 @@
+---
+title: PolymerTemplate, Creating a Simple Component Using the Template API
+order: 9
+layout: page
+---
+
+= PolymerTemplate, Creating a Simple Component Using the Template API
+
+*Usage of polymer based templates is deprecated since Vaadin 18 and we recommend using Lit based templates instead.*
```

Comment:
- “Polymer” should be capitalised
- Better break onto two separate sentences
- I guess LitElement should be mentioned with a complete name, otherwise it is confusing, as there are also `lit-html` templates

```suggestion
*Usage of Polymer based templates is deprecated since Vaadin 18. It is recommended to use LitElement based templates instead.*
```